jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -71,12 +71,10 @@ def mos_spec1d_parser(app, data_obj, data_labels=None):
 
     # Coerce into list-like object. This works because `Spectrum1D` objects
     #  don't have a length dunder method.
-    if not hasattr(data_obj, '__len__'):
+    if not hasattr(data_obj, "__len__"):
         data_obj = [data_obj]
     else:
-        data_obj = [Spectrum1D.read(x)
-                    if _check_is_file(x) else x
-                    for x in data_obj]
+        data_obj = [Spectrum1D.read(x) if _check_is_file(x) else x for x in data_obj]
```

Comment:
```suggestion
        data_obj = [Spectrum1D.read(x)
                    if _check_is_file(x) else x
                    for x in data_obj]
```

This change results in a line greater than 79 characters, and is less readable in my opinion.